Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift

```diff
@@ -65,14 +66,22 @@ class MSDistributeViewController: UITableViewController, AppCenterProtocol {
     self.edgesForExtendedLayout = []
   }
   
+  @IBAction func checkForUpdateSwitchUpdated(_ sender: UISwitch) {
+      UserDefaults.init().set(sender.isOn ? true : false, forKey: kSASAutomaticCheckForUpdateDisabledKey)
```

Comment:
Is `sender.isOn ? true : false` for handling the null case?

Reply:
@owlstronaut not sure what null case are you talking about?  `UISwitch` is not an optional.

Reply:
```suggestion
      UserDefaults.init().set(sender.isOn, forKey: kSASAutomaticCheckForUpdateDisabledKey)
```
I'm asking if this is the same thing @annakocheshkova